source/includes/mongosync-and-OIDC.rst

```diff
@@ -0,0 +1,4 @@
+Starting in ``mongosync`` 1.9.0, you can use ``mongosync`` to connect to
+MongoDB clusters with OpenID Connect (OIDC). For example, you can
```

Comment:
I think we should say that the supported OIDC providers are Microsoft Azure and Google Cloud Platform (instead of for example). Those are the only providers supported, so we need to make that clear